xbstrap/base.py

```diff
@@ -816,8 +816,11 @@ def traverse_graph(*, roots, visit, key=None):
     seen = set()
     stack = []
 
+    def default_mapping(n):
+        return
+
     if key is None:
-        key = lambda n: n
+        key = default_mapping
```

Comment:
`return n` would be correct here. But I think we should just disable the lint, the lambda is cleaner than the `def`.